src/components/Pages/Home.js

```diff
@@ -0,0 +1,38 @@
+import React from 'react';
+// import { Link } from 'react-router-dom';
```

Comment:
- [ ] it would be better to remove the unused commented code in order to keep the file clean and pro